hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/snapshot/SnapshotDiffManager.java

```diff
@@ -894,9 +894,9 @@ void generateSnapshotDiffReport(final String jobKey,
           new RocksDbPersistentMap<>(db, toSnapshotColumnFamily, codecRegistry,
               byte[].class, byte[].class);
       // Set of unique objectId between fromSnapshot and toSnapshot.
-      final PersistentMap<byte[], SnapshotDiffObject> objectIdToDiffObject =
+      final PersistentMap<byte[], byte[]> objectIdToDiffObject =
```

Comment:
nit: You can use Boolean directory here because there is not much saving on serialization and deserialization here. It is serialization when it is added `objectIdToDiffObject` and `deserialization` when reading it.

Reply:
done

Reply:
Added a BooleanCodec for the same.

Reply:
done